### ISSUE TITLE
Brief explanation about how to test/use scss files in 4.x

### ISSUE THREAD

**Issue by dariogcode** [drupalprojects/kalatheme#109]
Hi, as probably you know I'm trying to use the 4.x with sass features. I generated the sub-theme for 4.x and it work fine, but the new subtheme folder only contains the default .info file, template.php and the empty css. My question is if the subtheme generator is responsible to generate also a folder with the scss files or if this is something manual we should do.

I see a gemfile but not sure if this is used in combination with other tools. Any help is appreciated.

**Comment by bassettsj** [comment]
Hey there!
I would be happy help out here. The current subtheme generator is pretty limited in what it can do. We are looking at using Yeoman to help scaffold out more advanced subthemes. Would it be helpful to put up an empty template repo for the time being that can help get you going?

**Comment by dariogcode** [comment]
I don't know much about Yeoman but looks very promising!.

A template can help a lot. In my case I generated the theme without select a bootstrap librarty. Then I added a folder scss with files 

_bootstrap-variables.scss
styles.scss

And in styles.scss I imported bootstrap-variables and imported "selective" bootstrap scss

This will be my "template", but not sure if you're talking about something different?

**Comment by bassettsj** [comment]
@tilon My apollogies again on this. I have been up to my :eyes: in client work lately, haven't had time for anything else lately. I cleaned up scaffolding I used for another project. It has some great features and a nice README. Checkout the [kalamuna/kalatheme_sauce](https://github.com/kalamuna/kalatheme_sauce) for the time being on how to do these more advanced  things with kalatheme

Does that help?
